CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Changelog
+==========================
+This doc describes all notable changes to Diem Core and Diem Framework between.
```

Comment:
this sentence seems either not complete or mean something else? :P

Reply:
copy pasta from quip